modules.json

```diff
@@ -582,16 +582,25 @@
             "format": "json"
         }
     },
-	{
+	  {
         "filename": "get_alldeletedrevs",
         "docstring": "Demo of `alldeletedrevisions` module: List the all deleted revisions from User",
         "endpoint": "https://en.wikipedia.org/w/api.php",
         "params": {
             "action": "query",
             "list": "alldeletedrevisions",
             "adruser": "Mahesh",
-			"adrprop": "ids|user|comment",
-			"format": "json"
+			      "adrprop": "ids|user|comment",
+            "format": "json"
+        }
+    },
+    {
+        "filename": "rsd",
+        "docstring": "Demo of `Rsd` module: Get request to export an RSD schema.",
+        "endpoint": "https://en.wikipedia.org/w/api.php",
+        "params": {
+            "action": "rsd",
+			      "format": "json"
```

Comment:
Indentation. It's not usually visible at first glance, but check whether you are consistently using spaces or tabs in your schema. Mixing them is what results in the inconsistency.